Content guidelines (style guide improvement) (#9618)
ethereum/ethereum-org-website: src/content/contributing/style-guide/index.md

## Audience {#audience}

> - 60% of our website visitors do not own any Ether according to our survey responses

Reviewer: Made a separate note about "ether" capitalization... per our current style guide we use lowercase, but as I noted, the Yellow Paper consistently capitalizes this, so we may want to separately consider switching the casing here.

But in the meantime:
```suggestion
- 60% of our website visitors do not own any ether according to our survey responses
```

- "I want to understand the differences between private Ethereum chains, consortium chains, and the public Ethereum Mainnet"
- "I want to know the current status of Ethereum - how long has it been in production, how much usage it has, what's the direction of new development - to decide if I am confident to build my project on top of it"

</ExpandableCard>

## Best practices {#best-practices}

**Style**

- Focus on the advantages for the user instead of explaining technical details about the system
- Use active voice and clear, concise sentences that are easy to follow
- Break up longer chunks of text into smaller sections or paragraphs
- Consider using tables, bullet points or numbered lists instead of paragraphs
- Highlight (bold) key phrases to support scanning and skimming through the article

**Content ideas:**

- Use examples or real-life scenarios of the application of the technology to help illustrate complex concepts or ideas
- Explain how the idea can positively affect people now or in the future
- Create before Ethereum / after Ethereum comparison

> - Add step-by-step / how-to dos to take action

Reviewer: ```suggestion
- Add step-by-step how to take action
```
or maybe
```suggestion
- Add step-by-step instructions how to take action
```

- Include relevant statistics or graphs to strengthen the arguments
- Add calls to action
- Provide visual aids to explain the topic better

**Other**:

> - Limit the legth of the article up to 1000 words (1500 max)

Reviewer: ```suggestion
- Limit the length of the article up to 1000 words (1500 max)
```

- "I work at a business that is beginning an Ethereum related project, and want to learn more"
- "I want to understand the differences between private Ethereum chains, consortium chains, and the public Ethereum Mainnet"
- "I want to know the current status of Ethereum - how long has it been in production, how much usage it has, what's the direction of new development - to decide if I am confident to build my project on top of it"
</p>
</details>

## Best practices {#best-practices}

**Style**

> - [ ]  Focus on the advantages for the user instead of explaining technical details about the system

Reviewer 2: I think these will get rendered as tickable-boxes? Does it make sense to have those here?

Reviewer: Would have to double check this actually... not sure we have support for markdown checkboxes, since we don't actually use this syntax anywhere else in the repo right now

- [ ]  Use active voice and clear, concise sentences that are easy to follow
- [ ]  Break up longer chunks of text into smaller sections or paragraphs
- [ ]  Consider using tables, bullet points or numbered lists instead of paragraphs
- [ ]  Highlight (bold) key phrases to support scanning and skimming through the article

**Content ideas:**

- [ ]  Use examples or real-life scenarios of the application of the technology to help illustrate complex concepts or ideas
- [ ]  Explain how the idea can positively affect people now or in the future
- [ ]  Create before Ethereum / after Ethereum comparison
- [ ]  Add step-by-step / how-to dos to take action
- [ ]  Include relevant statistics or graphs to strengthen the arguments
- [ ]  Add calls to action
- [ ]  Provide visual aids to explain the topic better

**Other**:

- [ ]  Limit the legth of the article up to 1000 words (1500 max)

> - [ ]  Reduce the number of hyperlinks to approximately 5 per 1000 words (excluding further reading section at the bottom of the page or product listings)

Reviewer 2: also typo @ line 110: `legth` -> `length`

Many of the topics covered on ethereum.org are technically complex. To reduce confusion to the reader, terms should be used consistently. For example, don't cycle back-and-forth between proof-of-work and PoW at random.

> [Content standartization](/style-guide/content-standartization/) - Read more about proper usage of terminology and other aspects such as how to properly add an image, atribute etc.

Reviewer 2: ```suggestion
[Content standardization](/style-guide/content-standardization/) - Read more about proper usage of terminology and other aspects such as how to properly add an image, attribute etc.
```
Should also update the directory name `standartization` --> `standardization`. This would be mroe aligned with our preference for American English.
